docs/.prettierrc

```diff
@@ -0,0 +1,11 @@
+{
+  "printWidth": 80,
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": false,
+  "doubleQuote": true,
+  "trailingComma": "es5",
+  "bracketSpacing": true,
+  "jsxBracketSameLine": false,
+  "fluid": false
+}
```

Comment:
@zaceno Why do we need this? Not having this would be great.

Reply:
I don't think we need it, technically. I could remove it from from the PR. But, _I_ needed it while I was working because for some inscrutable reason the combination of my editor-plugins (VS Code + Prettier) with the .editorconfig and .eslint rules already in the repo, my editor started adding semicolons everywhere every time I saved. The only way I could figure out to solve it without removing any of the files already in the repo, was to also add this prettier config.

I thought since VSCode + Prettier extension is a pretty common set up it's best to leave it in until someone who has been working more with the website proper could consolidate all the config stuff. (Perhaps use only prettier and not editorconfig and eslint?)

But I'm fine taking it out of the PR as well. In that case maybe I'll add it to .gitignore so I can have it locally?

Reply:
I thought the `.eslint` was already removed by @loteoo in https://github.com/jorgebucaran/hyperapp/pull/920. I recall discussing this.

Reply:
I saw a conversation about it too, but no, it's still there: https://github.com/jorgebucaran/hyperapp/tree/master/docs

...I would toss it all myself but since I haven't been working on any of the rest of the website I thought that a bit presumptuous.